nautobot/docs/development/jobs/index.md

```diff
@@ -685,8 +679,6 @@ To skip writing a log entry to the database, set the `skip_db_logging` key in th
 
 Markdown rendering is supported for log messages, as well as [a limited subset of HTML](../../user-guide/platform-functionality/template-filters.md#render_markdown).
 
-+/- 1.3.4 "Log entry sanitization"
-    As a security measure, the `message` passed to any of these methods will be passed through the `nautobot.core.utils.logging.sanitize()` function in an attempt to strip out information such as usernames/passwords that should not be saved to the logs. This is of course best-effort only, and Job authors should take pains to ensure that such information is not passed to the logging APIs in the first place. The set of redaction rules used by the `sanitize()` function can be configured as [`settings.SANITIZER_PATTERNS`](../../user-guide/administration/configuration/settings.md#sanitizer_patterns).
```

Comment:
change to a `!!! warning` instead of removing?

Reply:
makes sense, done!